### ISSUE TITLE
Grid doesn't generate expected modifiers

### ISSUE THREAD

**Issue by johndjameson** [mvcss/mvcss#18]
Right now, the updated Grid component doesn't output `#of5`, `#of8`, or `#of10` modifiers.

We had these in the grid [before](https://github.com/mvcss/mvcss/blob/5ebd4845b70cf7ed7d49706c0b04ba71c0875ff0/components/_grid.sass#L83-L147), so we'll need to find a way to restore them.

I can think of two options:

1. Author them in the `g-device-type` mixin manually, like so:

  ```sass
  =g-device-type($namespace: '')

  // ...

  @if $g-columns >= 5
    @for $column from 1 to 5
      #{$g-selector}g-b--#{$namespace}#{$column}of5
        width: percentage($column / 5)

  // ...

  ```

2. Somehow alter the loop logic to generate the modifiers.

Ideas?

cc @mvcss/owners @mvcss/contributors

**Comment by drewbarontini** [comment]
The first option works for me, unless @dangodev (or someone else) has a way to modify the loop logic to include them. That'd be ideal, if we could do it.

**Comment by drewbarontini** [comment]
I also noticed that `$g-columns < 8` won't output `1of4`, `3of4`. You have to set it to `8` to get those.

**Comment by dangodev** [comment]
Ah. That’s differently from how I was using the grid. There’s not really an easy way to automate different denominators like 8, 10, and 12 all while keeping it efficient with `@ extend` (e.g., even though 8 and 12 have some overlap, some divisions like `5 / 8` don’t go in to 12, so they’re really separate grid systems).

One thing I can do is allow `$g-columns` to allow an array:

```
$g-columns: 8, 10, 12
```

Then it’ll stay efficient—only generating the math it needs to, without bloat (keep in mind that if you included `10`, then it’d also include `5` since the automator includes all perfect divisions of a number).

**Comment by dangodev** [comment]
Give a look-see and see if this is ideal behavior for you @drewbarontini @johndjameson: https://github.com/mvcss/mvcss/pull/19

**Comment by drewbarontini** [comment]
Fixed in #21
